### ISSUE TITLE
Quick pick should announce how many results are genered after pressing a character

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52542]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Screenreader,NVDA[2018.2],MAS1.3.1

**Repro Steps:**
1.Navigate to Status Bar and select "Select Language Mode" button.
2.Enter some data in the editor it shows some result related to entered data.
3.Observed that NVDA is not announcing the generated result.

**Actual:**
If the user enter letter "s" then all list which contains "s" will display.
But the nvda is reading only entered data of text editor.

**Expected:**
If the user enter letter "s" then all list which contains "s" will display.
then nvda should reads/announcing the how many results are generated after entering "s".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
User will not understand how many results are generated on entered data.

**MAS Reference:**
MAS1.3.1: [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_LangMode_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2122033/Accessibility.A11y_VSCode_LangMode_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
QuickPick usualy has hunderds if not thousands of results. I see no value in the user knowing what is that number. We could maybe read it out only if lower than some number.

I feel like this should be closed as designed
@bpasero let me know what you think

**Comment by isidorn** [comment]
In practice results come instantaniously. Thought we could experiment with an alert for the number of items. Might be noise. Let's assign to July

**Comment by bpasero** [comment]
@isidorn 

> In practice results come instantaniously.

Maybe for our pickers, but we cannot tell for extensions that use the picker.

**Comment by chrmarti** [comment]
Only the new API for pickers will allow for extensions to update the list at all (and with delay). I expect quick results to remain the common case. That might result in an announcement after each keystroke, if I understand the proposed change correctly.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Here Screenreader should announced the count, that how many numbers of reults are generated related to entered letter please refer the attached video too.

**Comment by chrmarti** [comment]
Using `"aria-live"="polite"`, that is read only when its text changes (here the number of results).

**Comment by isidorn** [comment]
Adding bug label so we verify this in endgame
